website/www/site/content/en/documentation/transforms/java/aggregation/distinct.md

```diff
@@ -33,22 +33,11 @@ values for each key.
 
 ## Examples
 
-**Example 1** Find the distinct element from a `PCollection` of `String`.
+**Example 1**: Find the distinct element from a `PCollection` of `String`.
```

Comment:
Don't remove current static example. MaAdd PGE as "Example 2: Find the distinct element from a `PCollection` of `Integer`"